README.md

```diff
@@ -61,14 +61,50 @@ Then, transform incoming params in your controller using `Decanter#decant`:
 
 ### Generators
 
-Decanter comes with generators for creating `Decanter` and `Parser` files:
+Decanter comes with custom generators for creating `Decanter` and `Parser` files:
+
+#### Decanters
 
 ```
 rails g decanter Trip name:string start_date:date end_date:date
+
+# Creates app/decanters/trip_decanter.rb:
+class TripDecanter < Decanter::Base
+  input :id, :integer
```

Comment:
I don't think this is included automatically?

Reply:
You're right. Removed both here and in the other example.

Reply:
You mean the `id` parser, right? You're correct - removed from this example as well as the other ✔️